packages/node_modules/@ciscospark/plugin-people/src/people-batcher.js

```diff
@@ -6,11 +6,14 @@ import {Batcher} from '@ciscospark/spark-core';
 
 /**
  * @class
+ * @ignore
```

Comment:
TIL: `@ignore`

Reply:
it's an alias for `@private`, but I'm trying to differentiate between things I don't want in the public docs and things that should not be documented. The whole people batcher class should be documented and used by internal code, it just doesn't belong on the developer site.